src/bindata.cr

```diff
@@ -1,3 +1,6 @@
+class BinDataVerificationException < Exception
```

Comment:
I feel like this should be under the `BinData` namespace
```
abstract class BinData
  class VerificationException < Exception
```

Reply:
Agreed.